### PR TITLE
Added support for Oracle UNPIVOT keyword.

### PR DESCRIPTION
**Pull request by losserfpv**
Added support for Oracle UNPIVOT keyword. Activated an already existing test file to test this feature: pivot01.sql. The test is successful.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.github.jsqlparser</groupId>
     <artifactId>jsqlparser</artifactId>
-    <version>3.1-SNAPSHOT</version>
+    <version>3.1-SNAPSHOT-ActFact-1</version>
```
Comment (wumpz) [inline pom.xml]: Please don't change the version.

File: src/main/java/net/sf/jsqlparser/expression/ExpressionVisitorAdapter.java [hunk 2]
```diff
@@ -14,19 +14,7 @@
...
-import net.sf.jsqlparser.statement.select.OrderByElement;
-import net.sf.jsqlparser.statement.select.Pivot;
-import net.sf.jsqlparser.statement.select.PivotVisitor;
-import net.sf.jsqlparser.statement.select.PivotXml;
-import net.sf.jsqlparser.statement.select.SelectExpressionItem;
-import net.sf.jsqlparser.statement.select.SelectItemVisitor;
-import net.sf.jsqlparser.statement.select.SelectVisitor;
-import net.sf.jsqlparser.statement.select.SubSelect;
-import net.sf.jsqlparser.statement.select.WithItem;
+import net.sf.jsqlparser.statement.select.*;
```
Comment (wumpz) [inline src/main/java/net/sf/jsqlparser/expression/ExpressionVisitorAdapter.java]: Actually the non star reference can be faster loaded and validated by a class loader. :) However it's a matter of taste.

File: src/main/java/net/sf/jsqlparser/util/deparser/SelectDeParser.java [hunk 3]
```diff
@@ -245,6 +249,21 @@ public void visit(Pivot pivot) {
         }
     }
 
+    @Override
+    public void visit(UnPivot unpivot) {
+        boolean showOptions = unpivot.getIncludeNullsSpecified();
```
Comment (wumpz) [inline src/main/java/net/sf/jsqlparser/util/deparser/SelectDeParser.java]: If possible use the visit methods to deparse.

File: pom.xml (hunk 1 above)
Comment (losserfpv) [reply pom.xml]: How can I fix this? Revert the change and create a new pull request?

Comment (wumpz) [reply pom.xml]: Just commit into your pull request. I will see the changes immediatly. I do always a Squash and Merge.

File: src/main/java/net/sf/jsqlparser/expression/ExpressionVisitorAdapter.java (hunk 2 above)
Comment (losserfpv) [reply src/main/java/net/sf/jsqlparser/expression/ExpressionVisitorAdapter.java]: I think it was automatically changed by IntelliJ

File: src/main/java/net/sf/jsqlparser/util/deparser/SelectDeParser.java (hunk 3 above)
Comment (losserfpv) [reply src/main/java/net/sf/jsqlparser/util/deparser/SelectDeParser.java]: I'm not sure how to change this. To me it just looks like how Pivot was implemented.

**Pull request merged**